src/coreclr/jit/redundantbranchopts.cpp

```diff
@@ -178,15 +178,123 @@ void Compiler::optRelopImpliesRelop(RelopImplicationInfo* rii)
 
     genTreeOps const oper = genTreeOps(funcApp.m_func);
```

Comment:
I assume it doesn't currently handle unsigned comparisons? (e.g. VNF_GT_UN)

Reply:
Correct, it does not handle them.